content/blog/how-to-fail-gracefully/index.md

```diff
@@ -0,0 +1,50 @@
+---
+date: "2021-01-05T15:00:00Z"
+description: "Empower your end-users to know the difference between a broken app and a data entry issue."
+featuredImage: visuals-JpTY4gUviJM-unsplash.jpg
+featuredImageCredit: "visuals"
+featuredImageLink: "https://unsplash.com/photos/JpTY4gUviJM"
+tags: []
+title: How to Fail Gracefully
+---
+
+_**Fail Gracefully!** Empower your users by providing them with applications that fail gracefully, or indicate clean error messaging without stopping them from getting the job done._
+
+# TL;DR
+
+If fail you must, then provide an accurate error message on the User Interface to guide users to resolve the issues.
+
+# What the heck does it mean?
+
+"We are unable to perform this operation. We are sorry for the inconvenience, please check back later.'' When throwing your computer out the window isn't an option, what else can you do? Nothing.
+
+Most of the time we move on, just work through the thing, and try it a different way. Maybe an incognito window, maybe clear our cache, maybe restarting our computer? Not everybody has an automated way to debug an issue. If they did, then software companies everywhere would cease to exist.
+
+
+# Support the User
+
+As Tech Support (yes, we are support, we create, we manage, we develop, we guide, but we support), we need to put ourselves back into the shoes of the User. We cannot always assume that the User is doing it wrong, because we build the applications for the User. The error checking in the code knows enough to display an error. At the very least, if the User is 'doing it wrong', then the error that we throw out should be smart enough to tell the User what happened. It is okay to be transparent- it fixes issues quicker, and builds trust with the user.
+
+What if throwing a catch-all error and saying 'open a ticket with support', we take the time to **at least** show the user what **could** be wrong. 
```

Comment:
I think a couple words are missing, maybe something like:
```suggestion
What if instead of throwing a catch-all error and saying 'open a ticket with support', we take the time to **at least** show the user what **could** be wrong. 
```